.coveragerc

```diff
@@ -16,14 +16,25 @@
 # specific language governing permissions and limitations
 # under the License.
 
-[report]
+[run]
+branch = True
+relative_files = True
+source = airflow
 omit =
-    tests/*
-    scripts/*
-    dev/*
```

Comment:
Why should we remove these?

Reply:
I explicit set `source = airflow` so there is no any any coverage outside of airflow module